### ISSUE TITLE
BV Errata for TAG calculations

### ISSUE THREAD

**Issue by HammerGS** [MegaMek/megamek#3385]
New Errata regarding BV Calculations a TAG units.

https://bg.battletech.com/forums/tactical-operations/tagguided-munitions-and-bv/

`BV is added to units with TAG. The Ammo BV is also added per piece of TAG capable equipment. So a unit that has two TAG-capable pieces of equipment, you have to apply the ammo BV twice.`

**Comment by SJuliez** [comment]
Which weapons exactly count as TAG? Our F_TAG flag contains e.g. the BA TAGs and C3MBS:
![image](https://user-images.githubusercontent.com/17069663/190844035-4d9bb674-8016-46a0-8ecd-17c06a3724be.png)

**Comment by SJuliez** [comment]
Does the TAG bonus also apply to ammo that the unit with TAG is carrying itself?

**Comment by SJuliez** [comment]
Does the C3 bonus work with the BV already modified by TAG or does it use the base BV?

**Comment by HammerGS** [comment]
Hi, got some clarifications from CGL

> Which weapons exactly count as TAG? Our F_TAG flag contains e.g. the BA TAGs and C3MBS: ![image](https://user-images.githubusercontent.com/17069663/190844035-4d9bb674-8016-46a0-8ecd-17c06a3724be.png)

Any type of TAG would trigger the errata.



> Does the TAG bonus also apply to ammo that the unit with TAG is carrying itself?

Yes




> Does the C3 bonus work with the BV already modified by TAG or does it use the base BV?

As far as the multi TAG answer goes, I'd only count double TAG, if it actually works as double TAG. Mentioning it, because I don't know how if there isn't a situation where multiple BAs can habe a TAG, but it is still only one possible attack and not 5

A clarification on stacking bonuses.
Then TAG first followed by C3. Everything that benefits from C3 gets taken u to account when calculating C3

**Comment by SJuliez** [comment]
Thanks! My third question was unclear. I was not speaking of C3M acting as TAG but of the C3 force bonus (add 5% of BV of the C3 unit to all) that comes after the TAG force bonus. So I assume the 5% include any TAG bonuses.

**Comment by HammerGS** [comment]
> Thanks! My third question was unclear. I was not speaking of C3M acting as TAG but of the C3 force bonus (add 5% of BV of the C3 unit to all) that comes after the TAG force bonus. So I assume the 5% include any TAG bonuses.

It's based of the modified amount.

**Comment by Logajam** [comment]
@HammerGS @SJuliez  Poking you two instead of making a new issue request.  I got annoyed enough at Homing ammo and TAG not being implimented correctly yet that I came looking.

In a nut shell.
A unit with TAG calculates its BV as if the TAG has BV equal to the total amount of Homing ammo BV on it's team.  (Can you home in on an allies TAG?  Then it would be your team plus allies).  Once that is done all other modifiers for BV come into play, like piloting skill and C3.
